src/Tooltip/TooltipDictionary.yaml

```
@@ -7231,6 +7231,18 @@ securityStepToolContext:
 securityStepToolPort:
     content: The TCP port to communicate with the scanner. You can specify this to override the default port for that scanner.
     width: "400"
+securityStepInstanceUsername:
+    content: Username that will be used to scan specified application instance.
+    width: "400"
+securityStepInstancePassword:
+    content: Password that will be used to scan specified application instance. In most cases this should be a secret.
+    width: "400"
+securityStepToolScanId:
+    content: The Scan ID of a tool specific individual scan. In the case Burp Enterprise scans this will be a part of URL in the enterprise UI.
```

```suggestion
    content: The Scan ID of a tool-specific individual scan. For Burp Enterprise scans, this will be a part of the URL in the enterprise UI.
```
Can we provide an example URL?

---

updated